### DISCUSSION TITLE
when is this condition ever not true?

### DISCUSSION THREAD

**Discussion by nektro** [libuv/libuv#4477] [Q&A] [answered]
https://github.com/libuv/libuv/blob/v1.48.0/include/uv/errno.h#L26

**Comment by saghul** [comment, marked as answer]
Haiku seems to already have negated errnos: https://github.com/libuv/libuv/commit/89cbbc895bb459c6ab5319fa86a32a9fecbf8744

**Reply by nektro** [reply]
wow. thanks for the info!
